### PR TITLE
Add notes from call on 2015-09-30

### PR DESCRIPTION
**Pull request by amirmc**
I'd appreciate a sanity check on the Irmin section (@samoht, @talex5) and also the version section (@samoht, @avsm, @dbuenzli).  I just want to check I didn't misunderstand what was discussed.

If there are no comments by 2pm(ish) I'll assume all is well and merge anyway.

### CODE REVIEW

File: tmpl/wiki/weekly/2015-09-30.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+Things have been cleaned up with the patch but it seems there have been some
+issues with Travis, which someone needs to take a closer look at.  Someone
+also needs to review and test the patch (Anil will do this, feedback from
+others also welcome). The examples should also be checked over.
+
+One thing we know is still missing as there's an opportunity to retain the
+shorthand flags we have for `--xen` and `--unix` as part of the configure step.
+ThomasG's not had a chance to look at that yet but would be great to keep them
+if we can.
```
Comment (samoht) [inline tmpl/wiki/weekly/2015-09-30.md]: there's also some integration with environment variable that have been done by Daniel: https://github.com/dbuenzli/cmdliner/commit/e82fd02842719167c3a04fc7ed0edf9a8036203c

File: tmpl/wiki/weekly/2015-09-30.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+#### AoB ####
+
+- Anil is trying to do more automated builds around mirage/mirage and using
+CISO to build on pushes.  Hopefully can retire is-mirage-broken when this is
+done.
+
+- Dave has been working on the `xl` files stuff and has a small roadmap with a
+few issues in it.  Will go through this in the background over time and slowly
+improve the deployment steps.
```
Comment (samoht) [inline tmpl/wiki/weekly/2015-09-30.md]: See https://github.com/mirage/mirage/issues/443

File: tmpl/wiki/weekly/2015-09-30.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+[brk-chg]: https://mirage.io/wiki/breaking-changes
+
+#### Irmin roadmap ####
+
+There are a number of things we can be doing with Irmin so there's a question
+of how we decide what exactly what we should work on next. It seems people
+have lists of things they'd like to get done but pulling these into a clearer
+plan would be helpful.
+
+ThomasL can compile Irmin to Javascript and and then call it from Javascript.
```
Comment (talex5) [inline tmpl/wiki/weekly/2015-09-30.md]: "and and"

The Irmin JS bindings are here: https://github.com/talex5/irmin-js
I know nothing about JS, so if someone can show me a better syntax for the examples, I'd love to hear it. Here's my attempt: https://github.com/talex5/irmin-js/blob/master/examples/tests.js

File: tmpl/wiki/weekly/2015-09-30.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+It's a bit ugly but can be done and talks more about 'repos' and 'branches'
+rather than 'stores'.  This might be something to make more consistent within
+Irmin.  Would also like to get rid of `create` and clean up the webserver.
+Built an [Irmin-based webserver][irmin-www] but doesn't have ability to push
+from the store.
```
Comment (talex5) [inline tmpl/wiki/weekly/2015-09-30.md]: "push to the server" and "pull from the server" are what I'm mostly missing.

**Comment by amirmc** [comment]
Thanks @samoht, @talex5! I'll merge as soon as CI is green.

**Pull request merged**